prime-router/src/main/kotlin/Receiver.kt

```diff
@@ -63,6 +63,10 @@ open class Receiver(
      * the date time would have the offset applied to it
      */
     val dateTimeFormat: DateUtilities.DateTimeFormat? = DateUtilities.DateTimeFormat.OFFSET,
+    /**
+     * Location of the custom configuration files for this receiver
+     */
+    val schemaFolderPath: String? = null
```

Comment:
Remove the path as well.  We will just use the schema name only.  The goal is that when we move the schemas to the DB then there will be no such thing as a path.  For not the path is part of the schema name.